main.go

```diff
@@ -199,12 +205,16 @@ func main() {
 	//
 	// Server
 	//
-
 	httpTimeout := 10 * time.Minute
 	maxHeaderBytes := 10 * (1024 * 1024)
 
+	if tlsCertificate != "" && tlsKey != "" {
+		logger.Info("Unable to leverage both Let's Encrypt and certificate/key flags at the same time - using the certificate/key pair...")
+		letsencrypt = false
+	}
+
 	// Plain text web server for use behind a reverse proxy.
-	if !letsencrypt {
+	if !letsencrypt && tlsCertificate == "" || tlsKey == "" {
```

Comment:
```suggestion
	if !letsencrypt || tlsCertificate == "" || tlsKey == "" {
```

Reply:
```
when:
  le == false 
  kp == true

(!le || kp) then pass (which we don't want, correct?)

when:
  le == false 
  kp == true

(!le && cert) then fail (which is what we want, no?)
```

So it should stay a **&&** and not be changed to a **||** - or am I missing it?